Flow.Launcher/Converters/QuerySuggestionBoxConverter.cs

```diff
@@ -44,6 +44,12 @@ public object Convert(object[] values, Type targetType, object parameter, Cultur
                     return string.Empty;
 
                 // When user typed lower case and result title is uppercase, we still want to display suggestion
+
+                string _suggestion = queryText + selectedResultPossibleSuggestion.Substring(queryText.Length);
+                if (String.IsNullOrEmpty(selectedResult.SuggestionText))
+                {
+                    selectedItem.Result.SuggestionText = _suggestion;
```

Comment:
@taooceros will this modifying result cause results comparison issue afterwards?

Reply:
Instead of using Result to hold the new constructed suggestion string, use [ResultViewModel](https://github.com/Flow-Launcher/Flow.Launcher/blob/2ba6ce9e874f80db739da756f79e8b6c3b759ed0/Flow.Launcher/ViewModel/ResultViewModel.cs#L145) instead. As a general we dont want to modify an object after it is constructed, otherwise it's hard to reason what state/information the object has.